### DISCUSSION TITLE
Update app with exported file

### DISCUSSION THREAD

**Discussion by fernandocalsa** [Budibase/budibase#6578] [Ideas]
There is a way of exporting/importing apps with the txt file we can export from an app. But, is there a way to update an app with one of those files, but without updating the users that have access to the app, neither the name/url?

We develop apps in one budibase deployment, and then export the app as a txt file, and upload it to the production budibase deployment, so every time we deploy a new change we have to give users access to the new app.

**Comment by maiducan** [comment]
You can export users to csv and import them in production.
IDs however wont be the same...

**Reply by fernandocalsa** [reply]
The problem is the association between users and the apps, and the data stored in Budibase tables, there is no a way to update an app with a txt file

**Reply by saaraazGit** [reply]
hey @fernandocalsa have you found any answers to this problem?
